applications/wordpress-microservice/templates/mariadb-deployment.yaml

```diff
@@ -12,11 +12,39 @@ spec:
   template:
     metadata:
       labels:
+       #{{- include "wordpress-microservice.selectorLabels" . | nindent 8 }}
```

Comment:
consider cleaning up some of the parts that are commented out and no longer needed if possible. this confused me for a bit.